src/core/api/track_choice_manager.ts

```diff
@@ -1058,10 +1071,16 @@ function getPeriodItem(
 function parseVideoRepresentation(
   { id, bitrate, frameRate, width, height, codec } : Representation
 ) : ITMVideoRepresentation {
-  return { id,
-           bitrate,
-           frameRate,
-           width,
-           height,
-           codec };
+  return { id, bitrate, frameRate, width, height, codec };
+}
+
+/**
+ * Parse audio Representation into a ITMAudioRepresentation.
+ * @param {Object} representation
+ * @returns {Object}
+ */
+function parseAudioRepresentation(
```

Comment:
Maybe not necessary to create such a function.

Reply:
It's more convenient considering that its called in two different `map`.
I can just do:
```ts
adaptation.representations.map(parseAudioRepresentation)
```
instead of:
```ts
adaptation.representations.map(({ id, bitrate, codec } => ({ id, bitrate, codec }));
```
two times.

Also it's the exact same principle than `parseVideoRepresentation`.